ci-operator/step-registry/openshift-qe/move-pods-infra/openshift-qe-move-pods-infra-commands.sh

```diff
@@ -0,0 +1,616 @@
+##############################################################################################
+if test ! -f "${KUBECONFIG}"
+then
+	echo "No kubeconfig, can not continue."
+	exit 0
+fi
+
+
+# Download jq
+if [ ! -d /tmp/bin ];then
+  mkdir /tmp/bin
+  export PATH=$PATH:/tmp/bin
+  curl -sL https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 > /tmp/bin/jq
+  chmod ug+x /tmp/bin/jq
+fi
+
+#Get Basic Infrastructue Architecture Info
+node_arch=$(oc get nodes -ojsonpath='{.items[*].status.nodeInfo.architecture}')
+platform_type=$(oc get infrastructure cluster -ojsonpath='{.status.platformStatus.type}')
+platform_type=$(echo $platform_type | tr -s 'A-Z' 'a-z')
+node_arch=$(echo $node_arch | tr -s " " "\n"| sort|uniq -u)
+
+######################################################################################
+#             CHANGE BELOW VARIABLE IF YOU WANT TO SET DIFFERENT VALUE               #
+######################################################################################
+#IF_MOVE_INGRESS IF_MOVE_REGISTRY IF_MOVE_MONITORING can be override if you want to disable moving
+#ingress/registry/monitoring
+
+export OPENSHIFT_PROMETHEUS_RETENTION_PERIOD=15d
+export OPENSHIFT_PROMETHEUS_STORAGE_SIZE=100Gi
+export OPENSHIFT_ALERTMANAGER_STORAGE_SIZE=2Gi
+
+case ${platform_type} in
+	aws)
+           #Both Architectures also need:
+           OPENSHIFT_PROMETHEUS_STORAGE_CLASS=gp3-csi
+           OPENSHIFT_ALERTMANAGER_STORAGE_CLASS=gp3-csi
+             ;;
+	gcp)
+           OPENSHIFT_PROMETHEUS_STORAGE_CLASS=ssd-csi
+	   OPENSHIFT_ALERTMANAGER_STORAGE_CLASS=ssd-csi
+             ;;
+	ibmcloud)
+	   OPENSHIFT_PROMETHEUS_STORAGE_CLASS=ibmc-vpc-block-5iops-tier
+	   OPENSHIFT_ALERTMANAGER_STORAGE_CLASS=ibmc-vpc-block-5iops-tier
+           OPENSHIFT_PROMETHEUS_STORAGE_CLASS=ibmc-vpc-block-5iops-tier
+	   OPENSHIFT_ALERTMANAGER_STORAGE_CLASS=ibmc-vpc-block-5iops-tier
+             ;;
+    	openstack)
+           OPENSHIFT_PROMETHEUS_STORAGE_CLASS=standard-csi
+           OPENSHIFT_ALERTMANAGER_STORAGE_CLASS=standard-csi
+             ;;
+	alibabacloud)
+	   OPENSHIFT_ALERTMANAGER_STORAGE_CLASS=alicloud-disk
+	   OPENSHIFT_PROMETHEUS_STORAGE_CLASS=alicloud-disk
+           OPENSHIFT_PROMETHEUS_STORAGE_CLASS=alicloud-disk
+           OPENSHIFT_ALERTMANAGER_STORAGE_CLASS=alicloud-disk
+             ;;
+
+	azure)
+	   #Azure use VM_SIZE as instance type, to unify variable, define all to INSTANCE_TYPE
+	   OPENSHIFT_PROMETHEUS_STORAGE_CLASS=managed-csi
+	   OPENSHIFT_ALERTMANAGER_STORAGE_CLASS=managed-csi
+             ;;
+        nutanix)
+	   #nutanix use VM_SIZE as instance type, to uniform variable, define all to INSTANCE_TYPE
+           OPENSHIFT_PROMETHEUS_STORAGE_CLASS=nutanix-volume
+           OPENSHIFT_ALERTMANAGER_STORAGE_CLASS=nutanix-volume
+	   ;;
+        vsphere|default)
+	  ;;
+	    
+	 *)
+	   echo "Un-supported infrastructure cluster detected."
+	   exit 1
+esac
+
+
+IF_MOVE_INGRESS=${IF_MOVE_INGRESS:=true}
```

Comment:
let's add these as environment variables in the ref file so that we could set them in the config to false if we want 
Example: https://github.com/openshift/release/blob/44003930b882788415c17c8b56d24c235c02ca02/ci-operator/step-registry/workers-scale/workers-scale-ref.yaml#L10

Reply:
@paigerube14 Fixed in new code, thank you for your suggestion!